templates/setup.cfg.tmpl

```diff
@@ -61,12 +61,12 @@ classifiers =
     Operating System :: OS Independent
 license = MIT
 license_files = LICENSE
+package_dir = {{SRC}}
```

Comment:
These [metadata] section is read by setuptools while running `setup` function in `setup.py`. Are you sure that adding `package_dir` here does not compromise anything in the building process?